src/SquidConfig.h

```diff
@@ -243,7 +243,7 @@ class SquidConfig
     } Addrs;
     size_t tcpRcvBufsz;
     size_t udpMaxHitObjsz;
-    wordlist *mcast_group_list;
+    SBufList mcast_group_list;
```

Comment:
Please do not add non-POD data members to `SquidConfig`. That class must be a POD because the global configuration object is accessed during program destruction.  `SquidConfig` is already documented to be a POD. Several existing data members violate this rule and that documentation[^1], but we should not make the problem more difficult to fix by adding new violations.

[^1]: We probably just did not know better when we added them.

Reply:
FYI: `SquidConfig` has contained other `SBufList` for a year now without problems. So has already been proven safe for this particular global usage.

FTR: the issues requiring strictly POD-only members have been resolved since Squid v4 (3b1439d03a1e603521ad95474483002238762eb2). Objects which use MEMPROXY allocation and are default constructable can be used, at least as members of existing globals now.